src/me/StevenLawson/TotalFreedomMod/Commands/Command_warn.java

```diff
@@ -40,13 +41,14 @@ public boolean run(CommandSender sender, Player sender_p, Command cmd, String co
 
         if (TFM_AdminList.isSuperAdmin(player))
         {
-            playerMsg(ChatColor.RED + "You can not warn admins");
+            playerMsg(ChatColor.RED + "You can not warn admins.");
             return true;
         }
 
         String warnReason = StringUtils.join(ArrayUtils.subarray(args, 1, args.length), " ");
-
-        playerMsg(ChatColor.RED + "[WARNING] " + warnReason);
+        
+        TFM_Util.bcastMsg(ChatColor.RED + sender.getName() + " - Warning " + player.getName());
+        playerMsg(player, ChatColor.RED + "[WARNING] " + warnReason + "\n Warned by " + sender.getName());
```

Comment:
@AvalancheYT playerMsg can be given two arguments, playerMsg(<player variable>, message).